PWGJE/EMCALJetTasks/macros/AddTaskEmcalJetEnergyScale.C

```diff
@@ -1,3 +1,3 @@
-PWGJE::EMCALJetTasks::AliAnalysisTaskEmcalJetEnergyScale *AddTaskEmcalJetEnergyScale(AliJetContainer::EJetType_t jettype, AliJetContainer::ERecoScheme_t recoscheme, AliVCluster::VCluUserDefEnergy_t energydef, double jetradius, Bool_t useDCAL, const char *namepartcont, const char *trigger, const char *suffix){
-  return PWGJE::EMCALJetTasks::AliAnalysisTaskEmcalJetEnergyScale::AddTaskJetEnergyScale(jettype, recoscheme, energydef, jetradius, useDCAL, namepartcont, trigger, suffix);
+PWGJE::EMCALJetTasks::AliAnalysisTaskEmcalJetEnergyScale *AddTaskEmcalJetEnergyScale(AliJetContainer::EJetType_t jettype, AliJetContainer::ERecoScheme_t recoscheme, AliVCluster::VCluUserDefEnergy_t energydef, double jetradius, Bool_t useDCAL, const char *namepartcont, const char *trigger, const char *suffix, const char *nametrackcont = "usedefault", const char *nameclustercont = "usedefault"){
+  return PWGJE::EMCALJetTasks::AliAnalysisTaskEmcalJetEnergyScale::AddTaskJetEnergyScale(jettype, recoscheme, energydef, jetradius, useDCAL, namepartcont, trigger, suffix, nametrackcont, nameclustercont);
```

Comment:
Also for the macros we need a way to not break existing train configurations. I think macros can have two functions with the same name but different parameter sets. The function with the matching parameter set will be called. We already use such approaches in several macros.

 Can you please add a second function with your new parameter list keeping the existing one as it is?